tabular/src/autogluon/tabular/models/tabular_nn/torch/tabular_torch_dataset.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, processed_array, feature_arraycol_map, feature_type_map, prob
                                             (options: 'vector', 'embed')
             problem_type (str): what prediction task this data is used for.
             labels (pd.Series): list of labels (y) if available
+            base_preds: 2D numpy array. Contains raw data of all base learners' predictions as columns
```

Comment:
If you end up keeping this same `TabularTorchDataset` for the aggregator networks, then should clarify this `base_preds` argument is specifically intended for aggregator networks that ensemble predictions from other models.

Also probably want to clarify what the format of these `base_preds` should be specifically for quantile regression applications (Does it only works for that quantile format or for regression/classification as well? if only for quantile, want to clarify that also).